src/fonduer/candidates/mentions.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, n_min=1, n_max=5, split_tokens=[]):
         """
         Ngrams.__init__(self, n_min=n_min, n_max=n_max, split_tokens=split_tokens)
 
-    def apply(self, session, doc):
+    def apply(self, doc):
         """Generate MentionNgrams from a Document by parsing all of its Sentences.
 
         :param session: The database session
```

Comment:
Session can be removed from the docstring if it's removed from the parameter list.